rancher-server/rancher-ha.tf

```diff
@@ -98,8 +98,13 @@ resource "rancher2_bootstrap" "admin" {
 }
 
 resource "rancher2_auth_config_github" "github" {
-  client_id             = var.github_client_id
-  client_secret         = var.github_client_secret
-  access_mode           = "restricted"
-  allowed_principal_ids = ["local://${data.rancher2_user.admin.id}", "github_user://3430214", "github_org://53273206", "github_team://3414845"]
+  count         = local.rancher2_auth_config_github_count
+  client_id     = var.github_client_id
+  client_secret = var.github_client_secret
+  access_mode   = "restricted"
+
+  allowed_principal_ids = ["local://${data.rancher2_user.admin.id}",
```

Comment:
This whole thing might be better as a concatenated list - the local admin user from data + whatever specific things the user specifies. That way they could define multiple users/orgs/teams/whatever.

Reply:
Good idea, done. Keeps existing signature allowing user/org/team to be specified as particular variables, but also allows an arbitrary list to be passed in as well